po4a-update-templates.sh

```diff
@@ -30,7 +30,7 @@ fi
 
 # Check if po4a is installed
 if ! [ -x "$(command -v po4a)" ] ; then
-    echo 'Error: please install po4a.' >&2
+    echo "Error: please install po4a." >&2
```

Comment:
Same as a `po4a-create-all-targets.sh` -- I'd question the use of double quotes in preference to single quotes.